AnnoDesigner/ViewModels/StatisticsViewModel.cs

```diff
@@ -237,8 +237,13 @@ public void ToggleBuildingList(bool showBuildingList, List<LayoutObject> placedO
 
             if (mode != UpdateMode.NoBuildingList && ShowBuildingList)
             {
-                var groupedBuildings = placedObjects.GroupBy(_ => _.Identifier);
-                var groupedSelectedBuildings = selectedObjects.Count > 0 ? selectedObjects.GroupBy(_ => _.Identifier) : null;
+                var groupedBuildings = placedObjects.GroupBy(_ => _.Identifier).ToList();
```

Comment:
Maybe change this to `groupedPlacedBuildings` to match the style of the other variable name?